protos/flyteidl/admin/schedule.proto

```diff
@@ -16,14 +16,20 @@ message FixedRate {
     FixedRateUnit unit = 2;
 }
 
+message CronSchedule {
+    string schedule = 1;
+    string offset = 2;
```

Comment:
one nit: can you add comments on what the expected format for schedule and offset should be?

Reply:
@katrogan Do we plan to support AWS cron as well in `CronSchedule.schedule`? If yes, I can revise the comment a bit.

Reply:
I think we should move away from the aws cron type (which is more work) and have admin handle converting to it in its aws specific scheduler handler. But that can be a separate issue and we can keep using the deprecated cron_expression in the meantime

Reply:
I see. The problem is the 5-fields cron syntax is weaker than the one used by AWS, so it's not gonna be an 100% mapping.